### DISCUSSION TITLE
Eine Übersicht von Rechtsbeiständen incl. Ortsbenennung die mit j-lawyer.org arbeiten

### DISCUSSION THREAD

**Discussion by hstr4** [jlawyerorg/j-lawyer-org#1620] [Ideas]
Eine Auflistung hier im Diskussionsbereich oder auf eine Unterseite die mit Pullrequests bearbeitet werden könnte und aufzeigt welche Rechtsbeistände ausschließlich mit freie Software wie j-lawyer ein Mandat bearbeiten würden wäre interessant. So könnten Menschen auf der Suche nach Rechtsbeistände welche freie Software wertschätzen schneller fündig werden.

Was hält @j-dimension von dieser Idee?

***************************************************************

- Dr. Michael Stehmann, Gladbacher Str. 3, 40764 Langenfeld, https://rechtsanwalt-stehmann.de/ , Lange Zeit FSFE-Mitglied, entwickelte bis 2018 eigene, freie Kanzlei-Software ( https://canzeley.de/ ), Fachanwalt für freie Softwarelizenzen, Vorstand von Freie Software Freunde e.V., Videovorträge vom Rechtsanwalt unter https://media.ccc.de/search?p=Michael+Stehmann

**Comment by j-dimension** [comment]
Die Idee ist grundsätzlich gut. Jedoch werden die allermeisten Anwender nicht willens / in der Lage sein, einen PR zu stellen oder GitHub zu bedienen... :-(

**Reply by hstr4** [reply]
Diese recht kleine GitHub-Hürde filtert automatisch Rechtsbeistände und ist vermutlich etwas gutes. Im Gegensaz zum dau-design von den meisten Rechtsbeistandsdatenbanken in Form von Webseiten die Geld für die Auflistung von Rechtsbeistände wollen wäre eine Auflistung hier kostenfrei und würde nur etwas Köpchen erfordern.

Da die verwendete Kanzlei-Software ggf. vom Rechtsbeistand nicht verraten werden sollte(recht auf informationelle selbstbestimmung), sollten Mandanten zuerst Rücksprache halten bevor Sie für den Rechtsbeistand so etwas öffentlich eintragen. Ausnahme natürlich wenn das bereits öffentliche Information ist und vom Rechtsbeistand längst selbstständig veröffentlicht wurde.

Vorschlag fürs Design einer Liste hier im Forum die von den Menschen durch Kopieren der bestehende Liste und ergänzen einer eigenen Information immer weiter gepflegt werden kann:
[Name][Anschrift][Webseite][Sonstige Informationen]
- Dr. Michael Stehmann, Gladbacher Str. 3, 40764 Langenfeld, https://rechtsanwalt-stehmann.de/ , Lange Zeit FSFE-Mitglied, entwickelte bis 2018 eigene, freie Kanzlei-Software ( https://canzeley.de/ ), Fachanwalt für freie Softwarelizenzen, Vorstand von Freie Software Freunde e.V., Videovorträge vom Rechtsanwalt unter https://media.ccc.de/search?p=Michael+Stehmann
- Ein anderer Rechtsbeistand, eine andere Anschrift, ...

**Reply by j-dimension** [reply]
Ich habe diese Diskussionsseite nun in der Haupt-Readme des Projektes verlinkt: https://github.com/jlawyerorg/j-lawyer-org

Werde gern bei neuen Kommentaren in dieser Diskussion die liste direkt im Ursprungsposting erweitern.

**Comment by dusmarcel** [comment]
Wir sind treue Fans :-) Rechtsanwalt Marcel Keienborg, Düsseldorf, Friedrich-Ebert-Straße 17, 40210 Düsseldorf, und Kollege
